src/support/unique_deferring_queue.h

```diff
@@ -23,6 +23,7 @@
 #ifndef wasm_support_unique_deferring_queue_h
 #define wasm_support_unique_deferring_queue_h
 
+#include <cassert>
```

Comment:
Why was this added?

Reply:
To fix compilation. The file uses assert, and by chance the other users had it included anyhow, so this wasn't a problem before.